pkg/orch/command_test.go

```diff
@@ -164,8 +164,8 @@ var expectedCommandScheduleTaskResponse = &ScheduledJobID{ScheduledJob: struct {
 
 func TestNewScheduleTaskOptions(t *testing.T) {
 
-	var optionsHour *ScheduleOptions = NewScheduleTaskOptions(time.Duration(1) * time.Hour)
-	var optionsMinutes *ScheduleOptions = NewScheduleTaskOptions(time.Duration(60) * time.Minute)
+	var optionsHour = NewScheduleTaskOptions(time.Duration(1) * time.Hour)
```

Comment:
out of interest why have you changed these lines?

Reply:
linting warnings - but can rollback?

Reply:
as long as the build still works. I thought the linter issues would have broken it but could be wrong.